### PR TITLE
Rename Kibana dashboard directories

### PR DESCRIPTION
**Pull request by ruflin**
Currently the dashboard directories are called 5.x and default. The naming default is a bit confusing as it is not clear what it means. In the current context it means the dashboards for all 6.x and 7.x releases. Unfortunately already now there are breaking changes in 7.x which means default does not work for both: https://github.com/elastic/beats/issues/5278

To allow us more flexibility the directory 5.x is renamed to 5 and the directory default is renamed to 6. This allows as to add a directory for 7 dashboards in the future. The reason I switched away from the `.x` naming is as this naming scheme also allows us to introduce for example 6.2 dashboards, meaning dashboards which only work with 6.2 and upwards in parallel to the 6 dashboards which work with all 6 releases.

TODO:
* [x] Rename all kibana directories in modules
* [x] Rename all kibana directories in all beats

@monica As a test I applied the change only to filebeat. To fully work all directories in our beats would have to be renamed. If we agree to move on with this, will update the PR with these changes too.

### CODE REVIEW

**Comment by ruflin** [comment]
The implementation could be changed, that `default` directory is a fallback. Meaning if the `6` or `6.2` directory is not found, it will try to find `default` directory.  This would make sure assuming we ship this with `6.1` that if someone keeps his `6.0`  directory with the `default` folder structure, it still works. A similar thing would have to be done for the `5.x` directory.

But as I we ship the dashboards with each release, I would expect people that load dashboards to use the one shipped with the binary so I do not see this as a breaking change.

**Comment by monicasarbu** [comment]
Yes, agree. This doesn't seem to be a breaking change, so we can add it to the `6.1` release.

File: libbeat/kibana/index_pattern_generator.go [hunk 1]
```diff
@@ -40,6 +40,7 @@ func NewGenerator(indexName, beatName, beatDir, beatVersion string, version comm
 
 // Create the Index-Pattern for Kibana for 5.x and default.
 func (i *IndexPatternGenerator) Generate() (string, error) {
+
```
Comment (ruflin) [inline libbeat/kibana/index_pattern_generator.go]: I hope @andrewkroh doesn't see this 🙈

Comment (andrewkroh) [reply libbeat/kibana/index_pattern_generator.go]: See what? 😉

**Comment by ruflin** [comment]
@monicasarbu @simitt This should be ready to go now.

@simitt Please also have a look as it will have an affect on apm and the naming of the directories there.

**Comment by ruflin** [comment]
@monicasarbu Could you please have a look at this one? I need to keep rebasing this as it affects a lot of files. Would be good to get this in rather soonish to make sure everything keeps working.

**Comment by ruflin** [comment]
@monicasarbu Rebase again on master and hopefully should go green now. Please merge if ok.

**Comment by ruflin** [comment]
jenkins, test it

**Pull request merged**